ui/components/app/wallet-overview/index.scss

```diff
@@ -16,30 +14,41 @@
     display: flex;
     gap: 4px;
     flex-direction: column;
-    align-items: center;
+    align-items: start;
     width: 100%;
   }
 
+  &__icon_button {
+    margin-top: 0 !important;
+  }
+
   &__buttons {
     display: flex;
     flex-direction: row;
-    height: 68px;
-    margin-bottom: 24px;
+    height: 100%;
+    margin-bottom: 16px;
+    padding: 0 16px;
   }
 
   &__portfolio_button {
-    display: flex;
-    flex-direction: row;
-    gap: 6px;
-    cursor: pointer;
-    align-items: center;
-    color: var(--color-primary-default);
+    margin-left: 8px;
+  }
+
+  &__portfolio_text {
+    font-family: "Euclid Circular B" !important;
+    font-size: 16px !important;
+    font-weight: 500 !important;
+    line-height: 22px !important;
+
+    @include design-system.screen-sm-max {
+      font-size: 14px !important;
+    }
   }
```

Comment:
Using static font styles I would consider a code smell we should endeavour to be using the Text component and props. All component library components have prop objects that allow access to the children components so you should be able to achieve this withough using these styles

```suggestion
```

Reply:
Removed here [52a8669](https://github.com/MetaMask/metamask-extension/pull/27161/commits/52a8669072e05aef09347df60e9ae1459dd543d3)